apel/common/unloader_utils.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+# Copyright 2021 UK Research and Innovation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+try:
+    import ConfigParser
+except ImportError:
+    # Renamed in Python 3
+    import configparser as ConfigParser
+
+log = logging.getLogger(__name__)
+
+RECORDS_PER_MESSAGE_MIN = 1
+RECORDS_PER_MESSAGE_DEFAULT = 1000
+RECORDS_PER_MESSAGE_MAX = 5000
+
+
+def check_records_per_message(cp):
```

Comment:
Maybe this function could be renamed to `get_records_per_message`? I feel something like `check_X` should return a boolean regarding whether X has passed or not.

If we did, the methods in the unit test should also be changed.